.github/workflows/buildcheck.yml

```diff
@@ -7,6 +7,15 @@ on:
 
 jobs:
   buildtest:
+
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix: # this needs to be an array. We should only list "known good/expected to work" vss-tools versions here
+        vss-tools-version: [master]
```

Comment:
@SebastianSchildt - I am thinking on how this possibly can be integrated with my ideas in #778 

What I think we need is one vss-tools version which is the "main" version for this particular commit. I.e. the version we shall use to generate release artifacts. It could be "master" for a regular commit (not intended to be released) but for release candidates and releases it should point to either a tag (like v4, v4.0, v4.0rc0, v5.1dev0 or similar) or a pypi release. And if we want a "release workflow" like in #778 we preferably need to have a static way to address the "official" vss-tools version (like an alias), so that we easily can fetch
 download from the folder "current" or similar.

You are better in github actions than I am so you most likely have some good ideas.